docs/content/about-iota/about-iota.mdx

```diff
@@ -85,4 +84,4 @@ Transactions on the IOTA network require [gas](./tokenomics/gas-in-iota.mdx) fee
 
 ## Consensus on IOTA
 
-IOTA uses a delegated proof-of-stake (DPoS) consensus mechanism to validate on-chain transaction blocks. [Validators](iota-architecture/validator-committee.mdx) on the IOTA network must secure a certain amount of IOTA tokens on the IOTA Mainnet to demonstrate their commitment to the network's security. This approach aligns the interests of all validators with those of IOTA users, ensuring an efficient and secure blockchain without the high energy demands of earlier blockchains.
+IOTA uses a delegated proof-of-stake (DPoS) consensus mechanism to validate on-chain transaction blocks. [Validators](iota-architecture/validator-committee.mdx) must secure a certain amount of IOTA tokens to participate in the network, incentivizing honest behavior and enhancing security. This approach ensures an efficient and secure blockchain while avoiding the high energy demands of Proof-of-Work (PoW) systems, which rely on computationally intensive mining.
```

Comment:
`participate in the network` -- validators not only "participate in the network", they `run the consensus protocol` (put a link to Mysticeti page).